Jenkinsfile

```diff
@@ -13,208 +13,363 @@
+                    agent { label 'armv8' }
+                    steps {
+                        script {
+                            def debugBuild = load ".jenkinsci/debug-build.groovy"
+                            if (!params.Linux && !params.MacOS) {
+                                debugBuild.doDebugBuild(true)
+                            }
+                            else {
+                                debugBuild.doDebugBuild()
+                            }
+                            if (BRANCH_NAME ==~ /(master|develop)/) {
+                                releaseBuild = load ".jenkinsci/release-build.groovy"
+                                releaseBuild.doReleaseBuild()
+                            }
+                        }
+                    }
+                    post {
+                        always {
+                            script {
+                                def cleanup = load ".jenkinsci/docker-cleanup.groovy"
+                                cleanup.doDockerCleanup()
+                                cleanWs()
+                            }
+                        }
+                    }
+                }
+                stage('MacOS'){
+                    when { expression { return params.MacOS } }
+                    agent { label 'mac' }
+                    steps {
+                        script {
+                            def coverageEnabled = false
+                            if (!params.Linux) {
+                                coverageEnabled = true
+                            }
                             def scmVars = checkout scm
                             env.IROHA_VERSION = "0x${scmVars.GIT_COMMIT}"
                             env.IROHA_HOME = "/opt/iroha"
-                            env.IROHA_BUILD = "/opt/iroha/build"
-                            env.IROHA_RELEASE = "${env.IROHA_HOME}/docker/release"
+                            env.IROHA_BUILD = "${env.IROHA_HOME}/build"
 
                             sh """
-                                ccache --version
-                                ccache --show-stats
-                                ccache --zero-stats
-                                ccache --max-size=1G
+                                /usr/local/bin/ccache --version
+                                /usr/local/bin/ccache --show-stats
+                                /usr/local/bin/ccache --zero-stats
+                                /usr/local/bin/ccache --max-size=5G
                             """
                             sh """
-                                cmake \
+                                /usr/local/bin/cmake \
                                   -DCOVERAGE=ON \
                                   -DTESTING=ON \
                                   -H. \
                                   -Bbuild \
                                   -DCMAKE_BUILD_TYPE=${params.BUILD_TYPE} \
                                   -DIROHA_VERSION=${env.IROHA_VERSION}
```

Comment:
this looks like a build code repetition.

Reply:
See my previous comment on `/usr/local/bin` on Mac